lib/view_component/preview.rb

```diff
@@ -62,6 +62,21 @@ def layout(layout_name)
         @layout = layout_name
       end
 
+      # Returns +true+ if the preview example has a template
+      def preview_example_template?(example)
+        return false unless example_exists?(example)
+
+        preview_example_template_path(example).present?
+      end
```

Comment:
Do we need both methods? We're always going to call them in combination. What if this method handled the defaulting we could simplify the whole thing.

Reply:
Ok sounds good, we can do that!